PowerShellTools/DebugEngine/HostUi.cs

```diff
@@ -264,6 +256,58 @@ public string ReadLine(string message, string name)
             return input;
         }
 
+        public int ReadChoice(string caption, string message, IList<ChoiceItem> choices, int defaultChoice)
+        {
+            if (String.IsNullOrEmpty(caption))

```

Comment:
Resolve this to comply with the codeing style here. https://github.com/dotnet/corefx/blob/master/Documentation/coding-style.md